Updating list topic (#421)
IgniteUI/indigo-design-docfx: en/components/list.md

## List Item Secondary Action

There are 4 interchangable List Item Secondary Actions that are listed below.

|                  |                                                                                                    |                                                                                                                                       |
| ---------------- | -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
| Badge            | <img class="responsive-img" src="../images/list_item_secondary.png" srcset="../images/list_item_secondary@2x.png 2x" />   |                                                                                                                                       |

> | Checkbox         | <img class="responsive-img" src="../images/list_item_secondary2.png" srcset="../images/list_item_secondary2@2x.png 2x" /> | Checkbox in a Secondary Action can not have a label, therefore, the Label Style is set to ~No Symbol and this setting should not be changed |

Reviewer: ```suggestion
| Checkbox         | <img class="responsive-img" src="../images/list_item_secondary2.png" srcset="../images/list_item_secondary2@2x.png 2x" /> | Checkbox in a Secondary Action can not have a label, therefore, the before and after labels are set to ~No Symbol and this setting should not be changed |
```

| Text and Icon            | <img class="responsive-img" src="../images/list_item_secondary3.png" srcset="../images/list_item_secondary3@2x.png 2x" /> |                                                                                                                                       |

> | Toggle             | <img class="responsive-img" src="../images/list_item_secondary4.png" srcset="../images/list_item_secondary4@2x.png 2x" /> | Switch in a Secondary Action can not have a label, therefore, the Label Style is set to ~ No Symbol and this setting should not be changed                                                                                                                 |

Reviewer: ```suggestion
| Toggle             | <img class="responsive-img" src="../images/list_item_secondary4.png" srcset="../images/list_item_secondary4@2x.png 2x" /> | Switch in a Secondary Action can not have a label, therefore, the before and after labels are set to ~ No Symbol and this setting should not be changed                                                                                                                 |
```

## Styling

The List comes with styling flexibility through the overrides available for background color and the different elements used in the List Items, such as icons and text, as well as components like Avatar, Badge, Checkbox, Icon, Progress, Switch, etc. with their own styling capabilities.

<img class="responsive-img" src="../images/list_styling.png" srcset="../images/list_styling@2x.png 2x" />

## List Preset

In addition, you can find a variety of Presets you can choose from for quick list creation.

> 

Reviewer: Assuming that you want to tell about the list patterns let's remove this from here and @sdimchevski will describe all list presets available as patterns in that topic.

> ## List Preset
> 
> In addition, you can find a variety of Presets you can choose from for quick list creation.
> 

Reviewer: ```suggestion
```

- [Progress](progress.md)
- [Switch](switch.md)
- [Lists Pattern](../patterns/lists.md)

>   <div class="divider--half"></div>
> 

Reviewer: ```suggestion
```
